src/app/component/ProductReviewForm/ProductReviewForm.style.scss

```diff
@@ -0,0 +1,168 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.ProductReviewForm {
+    &-Title {
+        margin-bottom: 0;
+    }
+
+    &-ProductName {
+        font-weight: 700;
+        font-size: 2rem;
+
+        @include mobile {
+            font-size: 1.8rem;
+        }
+    }
+
+    .Form {
+        max-width: 500px;
+
+        @include mobile {
```

Comment:
Are mobile wider than 500px? No need to reset it. Also, better use `100%` or `initial`, not `none`

Reply:
Currently mobile breakpoint is set at 767px, will adjust the max-width value though

Reply:
Then it is fine.